### PR TITLE
rails forms project

### PR DESCRIPTION
**Pull request by karthykarthick**
in this rails tutorial, we implemented forms project

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @karthykarthick 
### You have done well so far :clap: However, you still have a few issues to solve.
- [x] We need a commented code for `form` and `form_tag`
- [x] Add `rails db:create` as part of the project installation in the readme file
### Make the changes and submit for another code review. Good luck.
- Happy coding!

**Review by rOluochKe — CHANGES_REQUESTED** [review]
### Kindly the change from the previous review. Good luck.

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -33,3 +33,51 @@
...
+  <%# </p>  %>
+   <%# <p> Last added Users</p> %>
+
+    <%# <ul id="users"> %>
+  <%# <% @users.each do |user| %> %>
+   
+        <%# <%= 'Username = ' + user.username  %><br> %>
+        <%# <%= 'Email = ' + user.email  %><br>      %>
+          <%# <%= link_to "Edit User", edit_user_path(user.id) %> %>
+          <%# <br> <br>  %>
```
Comment (lucasmazo23) [inline app/views/users/new.html.erb]: The way the code is commented out shows me some closing tags when I open the form in my browser
image
image

**Review by lucasmazo23 — APPROVED** [review]
# Project approved 🚀

### Great job!
### Your project looks great!
### You can go ahead and merge this pull request!
### Happy coding! 💻

## Lucas Mazo

---

I'm approving this project because you did the change that was required, but please resolve the visual bug I mentioned in my comment below  before merging the pull request.

**Pull request merged**